tests/test_kubectl.py

```diff
@@ -0,0 +1,33 @@
+
+import pytest
+from pytest_container import DerivedContainer
+from pytest_container.container import ContainerLauncher
+from pytest_container.runtime import OciRuntimeBase
+
+from bci_tester.data import KUBECTL_CONTAINERS
+
+
+@pytest.mark.parametrize("ctr_image", KUBECTL_CONTAINERS)
+def test_kubectl_binary(
+    container_runtime: OciRuntimeBase,
+    pytestconfig: pytest.Config,
+    ctr_image: DerivedContainer,
+) -> None:
+    kubectl_ctr = DerivedContainer(
+        base=ctr_image,
+        custom_entry_point="/bin/bash",
+    )
+
+    with ContainerLauncher.from_pytestconfig(
+        kubectl_ctr, container_runtime, pytestconfig
+    ) as kubectl_launcher:
+        kubectl_launcher.launch_container()
+        kubectl_con = kubectl_launcher.container_data.connection
```

Comment:
You don't have to do the manual launching yourself here, just use the `container` or `auto_container` fixture.